File: 1010. Pairs of Songs With Total Durations Divisible by 60.cpp
class Solution {
public:
    int numPairsDivisibleBy60(vector<int>& time) {
        vector<int> c(65,0);
        int ans=0;
        for(int i=0;i<time.size();i++){
            time[i]%=60;
            ans+=c[(60-time[i])%60];
            c[time[i]]++;
        }
        return ans;
    }
};

File: 1010. Pairs of Songs With Total Durations Divisible by 60.go
func numPairsDivisibleBy60(time []int) int {
    var c [65]int
    var ans int
    for i := range time{
    	time[i]%=60
    	ans+=c[(60-time[i])%60]
    	c[time[i]]++
    }
    return ans
}